### PR TITLE
docs: fix installation and templates

### LINKED ISSUE #37: Wrong commands in the post install notes
The post install notes has 2 wrong command:
- Bad label value in the kubectl get pods command and does not return with the wanted pod name
- The port forward url protocol is http in the echo command while just https works.

### PR DESCRIPTION
**Pull request by ramizpolic**
## Overview

Fixes #37

## Notes for reviewer

After merge, we should rollout a new release. The current tag does not work seamlessly since it requires additional configuration to specify proper bank-vaults image.

### CODE REVIEW

File: vault/README.md [hunk 1]
```diff
@@ -36,11 +31,27 @@ helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault
 To install the chart backed with a Consul cluster, use the following:
 
 ```bash
-helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault --set vault.config.storage.consul.address="myconsul-svc-name:8500",vault.config.storage.consul.path="vault"
+helm install vault oci://ghcr.io/bank-vaults/helm-charts/vault \
+--set vault.config.storage.consul.path="vault"
```
Comment (akijakya) [inline vault/README.md:35]: `\` is missing from the end of this line

File: vault/README.md [hunk 2]
```diff
@@ -6,22 +6,17 @@ A tool for secrets management, encryption as a service, and privileged access ma
 
 This directory contains a Kubernetes Helm chart to deploy a Vault server. For further details of how are we using Vault read this [post](https://banzaicloud.com/blog/oauth2-vault/).
 
-## Prerequisites Details
+**Requirements**
```
Comment (akijakya) [inline vault/README.md:9]: What is the reason for replacing the headers with bold text?

File: vault/README.md (hunk 1 above)
Comment (ramizpolic) [reply vault/README.md]: nice catch!

File: vault/README.md (hunk 2 above)
Comment (ramizpolic) [reply vault/README.md]: looked too flash while it only contained a single item, reduced the size H3

**Review by akijakya — APPROVED** [review]
🚀

**Pull request merged**